helm/alfresco-content-services/templates/config-repository.yaml

```
@@ -107,6 +94,21 @@ data:
       -Djavax.net.ssl.keyStore=/var/run/secrets/java.io/keystores/keystore.jks -Djavax.net.ssl.keyStorePassword=changeit
       {{- end }}
       {{- end }}
+      {{- if .Values.mail.host }}
+      -Dmail.encoding={{ .Values.mail.encoding }}
+      -Dmail.host={{ .Values.mail.host }}
+      -Dmail.port={{ .Values.mail.port }}
+      -Dmail.protocol={{ .Values.mail.protocol }}
+      -Dmail.username={{ .Values.mail.username }}
```

```suggestion
      {{- printf('-Dmail.username=%s' .Values.mail.username) | default "" }}
```

As we can't really put a default inthe `values.yaml` file